pkg/plugins/plugins.go

```diff
@@ -96,7 +108,7 @@ func (p PluginDTO) IncludedInSignature(file string) bool {
 	}
 
 	// permit when no signed files (no MANIFEST)
-	if p.SignedFiles == nil {
+	if p.SignedFiles == nil { //TODO only accept unsigned if it is part of authorizer
```

Comment:
Guess this might indicate that reading plugin files needs to be a separate service or what do you have in mind?

Reply:
I'm thinking that we will basically remove this field and have an abstraction around Files which will exist on the model - this will basically be made up of the "signed files" / whatever is inside the plugin directory. Will probably be easier to illustrate this in some code (soon).

Can remove this TODO in that case

Reply:
Okay :+1: Fine to keep the TODO I think. Was just curious if this potentially creates a problem until fixed, i.e. always allows unsigned plugins to run or something?